### PR TITLE
A few suggestions with the contact update

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Contribution guideline with the updates on code of conduct, and contact info. A few suggestions on the language.

### List of changes proposed in this PR (pull-request)

* [x] One more sentence on Code of Conduct
* [x] Contact and Turing Way Twitter link updates

### What should a reviewer concentrate their feedback on?

- [x] Everything looks ok?

### Acknowledging contributors

- [x] Contributors are listed in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,12 +45,11 @@ Been here before? Already know what you're looking for in this guide? Jump to th
 ## Joining the community
 
 The Turing Way is a community-oriented and -led project.
-We therefore require that all our members and their contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
+We therefore require that all our members and their contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**. Please familiarize yourself with the [Code of Conduct](CODE_OF_CONDUCT.md) that lists the expected behaviours and our reporting guidelines.
```
Comment (KirstieJane) [inline CONTRIBUTING.md]:
```suggestion
We therefore require that all our members and their contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
Please familiarize yourself with the [Code of Conduct](CODE_OF_CONDUCT.md) that lists the expected behaviours and our reporting guidelines.
```
My on brand pedantry as usual 🙄

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -41,16 +45,15 @@ Been here before? Already know what you're looking for in this guide? Jump to th
...
 
 ## Inclusivity
 
-This project aims to be inclusive to people from all walks of life and to all research fields.
-This should be taken into account in contributions.
+This should be taken into account in contributions that the Turing Way aims to be inclusive to people from all walks of life and to all research fields.
 
-The following are examples of inclusive actions that we encourage from contributors to the Turing Way:
+In addition to the Code of Conduct, we encourage intentional inclusive actions from contributors to the Turing Way. Here are a few examples of such actions:
```
Comment (KirstieJane) [inline CONTRIBUTING.md]:
```suggestion
In addition to the Code of Conduct, we encourage intentional inclusive actions from contributors to _The Turing Way_.
Here are a few examples of such actions:
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -66,9 +69,10 @@ There are many ways to get in touch with the Turing Way team!
     Open a new issue!
     (See our [Where to start: issues](#where-to-start-issues) section below.)
 - Subscribe to our [mailing list](https://tinyletter.com/TuringWay) with which we send monthly project updates.
-- Check out the [#TuringWay](https://twitter.com/search?q=%23TuringWay&src=typed_query) hashtag on Twitter.
-- You can contact the PI of the Turing Way project - Kirstie Whitaker - by email at kwhitaker@turing.ac.uk.
-- You can also contact members of the Turing Way team through their preferred ways of communication [here](https://github.com/alan-turing-institute/the-turing-way/blob/master/ways_of_working.md).
+- Follow [@turingway on Twitter](https://twitter.com/turingway).
```
Comment (KirstieJane) [inline CONTRIBUTING.md]:
```suggestion
- Follow [@turingway](https://twitter.com/turingway) on Twitter.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -66,9 +69,10 @@ There are many ways to get in touch with the Turing Way team!
...
+- You can contact the Project Lead of the Turing Way project - Kirstie Whitaker - by email at kwhitaker@turing.ac.uk.
```
Comment (KirstieJane) [inline CONTRIBUTING.md]:
```suggestion
- You can contact the Project Lead of _The Turing Way_ project - Kirstie Whitaker - by email at [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -66,9 +69,10 @@ There are many ways to get in touch with the Turing Way team!
...
+- You can contact the Community Manager of the Turing Way project - Malvika Sharan - by email at masharan@turing.ac.uk.
```
Comment (KirstieJane) [inline CONTRIBUTING.md]:
```suggestion
- You can contact the Community Manager of _The Turing Way_ project - Malvika Sharan - by email at [masharan@turing.ac.uk](mailto:masharan@turing.ac.uk).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -66,9 +69,10 @@ There are many ways to get in touch with the Turing Way team!
...
+- You can also contact other members of the Turing Way through their preferred ways of communication [here](https://github.com/alan-turing-institute/the-turing-way/blob/master/ways_of_working.md).
```
Comment (KirstieJane) [inline CONTRIBUTING.md]: What do you think about removing this? I feel like it is so out of date that it probably needs to be archived now...

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -41,16 +45,15 @@ Been here before? Already know what you're looking for in this guide? Jump to th
...
 
 The Turing Way is a community-oriented and -led project.
-We therefore require that all contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**.
+We therefore require that all our members and their contributions **adhere to our [Code of Conduct](CODE_OF_CONDUCT.md)**. Please familiarize yourself with the [Code of Conduct](CODE_OF_CONDUCT.md) that lists the expected behaviours and our reporting guidelines.
 
 ## Inclusivity
 
-This project aims to be inclusive to people from all walks of life and to all research fields.
-This should be taken into account in contributions.
+This should be taken into account in contributions that the Turing Way aims to be inclusive to people from all walks of life and to all research fields.
```
Comment (KirstieJane) [inline CONTRIBUTING.md]:
```suggestion
_The Turing Way_ aims to be inclusive to people from all walks of life and to all research fields.
These intentions are manifest in the contributions that we make to the project.
```

I felt like your original version was burying the lede of what the Turing Way is aiming to do. But my second sentence is pretty jargony....feel free to adjust.

File: CONTRIBUTING.md (hunk 6 above)
Comment (malvikasharan) [reply CONTRIBUTING.md]: I think it really depends on the preference of other members who have strong ties with the project and still active contributors.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -50,7 +50,8 @@ Please familiarize yourself with the [Code of Conduct](CODE_OF_CONDUCT.md) that
 
 ## Inclusivity
 
-This should be taken into account in contributions that the Turing Way aims to be inclusive to people from all walks of life and to all research fields.
+_The Turing Way_ aims to be inclusive to people from all walks of life and to all research fields.
+These intentions are manifest in the contributions that we make to the project.
```
Comment (malvikasharan) [inline CONTRIBUTING.md]:
```suggestion
These intentions must be reflected in the contributions that we make to the project.
```

File: CONTRIBUTING.md (hunk 6 above)
Comment (KirstieJane) [reply CONTRIBUTING.md]: Lets leave it for now and maybe come back to it in the future :)

**Pull request merged**